src/styles/pages/_connectPage.scss

```
@@ -0,0 +1,41 @@
+.msteams-connect {
+    margin-left: 30px;
+    margin-top: 45px;
+
+    &__title {
+        height: 28px;
+        font-weight: 600;
+        font-size: 30px;
+        line-height: 28px;
+    }
+
+    &__description {
+        width: 746px;
+        height: 40px;
+        margin-top: 30px;
+        font-weight: 400;
+        font-size: 14px;
+        padding-bottom: 65px;
+        line-height: 20px;
+    }
+
+    &__button {
+        font-weight: 600;
+        padding: 6px 18px;
+        color: rgb(255, 255, 255);
+        box-shadow: var(--button-shadow);
```

```suggestion
        box-shadow: $button-shadow;
```
see you may need to import `_variable.scss` here